dev-cpp/gtest/gtest-9999.ebuild

```diff
@@ -1,10 +1,10 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 # Python is required for tests and some build tasks.
-PYTHON_COMPAT=( python3_{7,8,9} )
+PYTHON_COMPAT=( python3_{7..10} )
```

Comment:
ditto

Reply:
I assume you mean `8..10`, as suggested above.